resources/views/docs/1/the-basics/menu-bar.md

```diff
@@ -223,16 +223,16 @@ window events to the `nativephp` broadcast channel.
 
 To learn more about NativePHP's broadcasting capabilities, please refer to the [Broadcasting](/docs/digging-deeper/broadcasting) section.
 
-### Menu Bar Opened
+### MenuBarShown
```

Comment:
Maybe we put these names in code blocks?

```suggestion
### `MenuBarShown`
```

Reply:
Yeah, that could be a nice differentiator. Let's try.